src/ansys/sherlock/core/launcher.py

```diff
@@ -33,8 +33,13 @@ def _is_port_available(host=LOCALHOST, port=SHERLOCK_DEFAULT_PORT):
 
 
 def launch_sherlock(
-    host=LOCALHOST, port=SHERLOCK_DEFAULT_PORT, single_project_path="", sherlock_cmd_args=""
-):
+    host: str = LOCALHOST,
+    port: int = SHERLOCK_DEFAULT_PORT,
+    release_number: int = None,
+    single_project_path: str = "",
+    sherlock_cmd_args: str = "",
+    year: int = None,

```

Comment:
I recommend having the parameter for release number just after the year because they are used together.